docs/en/user_guides/5_visualization.md

```diff
@@ -1,67 +1,72 @@
-# Learn about Visualization
+# 学习如何进行可视化操作
 
-## Local Visualization
+## 本地可视化
 
-This section will present how to visualize the detection/tracking results with local visualizer.
+本节将介绍如何使用本地可视化工具可视化检测/跟踪结果。
 
-If you want to draw prediction results, you can turn this feature on by setting `draw=True` in `TrackVisualizationHook` as follows.
+如果你想绘制预测结果，你可以通过在`TrackVisualizationHook`中设置`draw=True`打开这个功能，如下所示。
```

Comment:
Please note that the one blank space between English & numbers and Chinese and check all the contents.

Reply:
Thanks for guidance. I have modified and submitted it to `dev-1.x`